Dockerfile

```diff
@@ -26,4 +27,6 @@ RUN pip install -r /app/dev-requirements.txt
 
 COPY . /app
 
+RUN pip install -e .
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. -->
**question:** I probably missed this in a previous PR, but what is this third pip install for?

Reply:
this installs the server package locally (seen in setup.py)